contracts/ethereum/src/PaymentRegistry.sol

```diff
@@ -22,13 +22,15 @@ contract PaymentRegistry is Initializable, OwnableUpgradeable, UUPSUpgradeable {
     event ModifiedZKSyncEscrowAddress(address newEscrowAddress);
     event ModifiedStarknetEscrowAddress(uint256 newEscrowAddress);
     event ModifiedStarknetClaimPaymentSelector(uint256 newEscrowClaimPaymentSelector);
-    event ClaimPayment(TransferInfo transferInfo);
```

Comment:
Why was this event removed?
I understand emitting one event per order claimed in a batch might be too much, but an event emitted after a whole batch is successfully claimed is appropriate

Reply:
There are three options as I see it regarding this event.

1. Emit one event per order fulfilled (waste of gas IMO)
2. Emit a different event (e.g ClaimPaymentBatch) that includes array of order_ids (we could also add destination addresses & amounts).
3. Remove event alltogether for consistency

I think you are suggesting we do 2, I agree it might be best

Reply:
Solution 2 implemented at 717aaa4cd6f2d57f042b98c83d275de0f6ef3fb3